Dockerfile

```
@@ -12,16 +16,19 @@ RUN npm ci --ignore-scripts
 COPY data/openclaw/workspace/projects/mission-control/frontend/ ./
 RUN npm run build
 
-# --- Stage 2: Unified runtime ---
+# --- Stage 3: Unified runtime ---
 FROM ghcr.io/openclaw/openclaw:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)